CONSOLE-SWITCH-FLOW-ANALYSIS.md

```
@@ -0,0 +1,385 @@
+
+### 3. **Parent loader not saved**
+fixed
+
+### 4. **Saved loader port not updated**
+fixed
+
+### 5. **No wait for USB enumeration**
+fixed
+
+### 6. **Improv test timing**
+fixed
```

markdownlint-cli2 (0.20.0)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)